### ISSUE TITLE
Release verbosity

### ISSUE THREAD

**Issue by albireox** [sdss/marvin#92]
In the spirit of tuning Marvin's verbosity down, does it make sense to have the warning

```
INFO: No release version set. Setting default to MPL-5
```

every time that one imports Marvin? Especially taking into account that there is currently no way of setting the release before importing marvin.

**Comment by bretthandrews** [comment]
When you import a python package it doesn't print its version by default (I
understand that this is somewhat different), so perhaps we shouldn't print
anything.

**Comment by havok2063** [comment]
it's not the python package, it's the version of the data Marvin is using.   Without this people will not know what version they are using unless they ask for it.  And we know people will not do that, or know to do that.  I think it's useful to see what the version of the data is being set to on startup.

**Comment by bretthandrews** [comment]
Yeah, I was just trying to use an analogy. Perhaps you're right that people
won't know how to check or will make wrong assumptions, so we should tell
them up front.
